### ISSUE TITLE
peg-syntax-ext broken?

### ISSUE THREAD

**Issue by fan-tom** [kevinmehall/rust-peg#203]
Cargo.toml
```toml
[dependencies]
...
peg-syntax-ext = "0.5"
```

```bash
$rustc --version
rustc 1.37.0-nightly (433a46781 2019-06-28)
```

Compilation error:
```
error[E0432]: unresolved import `syntax::fold`
  --> /Users/alexey/.cargo/registry/src/github.com-1ecc6299db9ec823/peg-syntax-ext-0.5.7/lib.rs:18:13
   |
18 | use syntax::fold::Folder;
   |             ^^^^ could not find `fold` in `syntax`

error: cannot find macro `panictry!` in this scope
  --> /Users/alexey/.cargo/registry/src/github.com-1ecc6299db9ec823/peg-syntax-ext-0.5.7/lib.rs:80:28
   |
80 |     while let Some(item) = panictry!(p.parse_item()) {
   |                            ^^^^^^^^

error: cannot find macro `panictry!` in this scope
  --> /Users/alexey/.cargo/registry/src/github.com-1ecc6299db9ec823/peg-syntax-ext-0.5.7/lib.rs:93:39
   |
93 |     let arg = cx.expander().fold_expr(panictry!(parser.parse_expr()));
   |                                       ^^^^^^^^

error[E0425]: cannot find function `IdentTT` in module `syntax::ext::base`
  --> /Users/alexey/.cargo/registry/src/github.com-1ecc6299db9ec823/peg-syntax-ext-0.5.7/lib.rs:27:32
   |
27 |             syntax::ext::base::IdentTT(Box::new(expand_peg_str), None, false));
   |                                ^^^^^^^ not found in `syntax::ext::base`

error[E0425]: cannot find function `IdentTT` in module `syntax::ext::base`
  --> /Users/alexey/.cargo/registry/src/github.com-1ecc6299db9ec823/peg-syntax-ext-0.5.7/lib.rs:31:32
   |
31 |             syntax::ext::base::IdentTT(Box::new(expand_peg_file), None, false));
   |                                ^^^^^^^ not found in `syntax::ext::base`

error[E0599]: no method named `fold_expr` found for type `syntax::ext::expand::MacroExpander<'_, '_>` in the current scope
  --> /Users/alexey/.cargo/registry/src/github.com-1ecc6299db9ec823/peg-syntax-ext-0.5.7/lib.rs:93:29
   |
93 |     let arg = cx.expander().fold_expr(panictry!(parser.parse_expr()));
   |                             ^^^^^^^^^

error[E0609]: no field `span` on type `syntax::parse::parser::Parser<'_>`
  --> /Users/alexey/.cargo/registry/src/github.com-1ecc6299db9ec823/peg-syntax-ext-0.5.7/lib.rs:99:44
   |
99 |                         cx.span_err(parser.span,
   |                                            ^^^^ unknown field
   |
   = note: available fields are: `sess`, `token`, `prev_span`, `recurse_into_file_modules`, `root_module_name`, `cfg_mods`

error[E0609]: no field `span` on type `syntax::parse::parser::Parser<'_>`
   --> /Users/alexey/.cargo/registry/src/github.com-1ecc6299db9ec823/peg-syntax-ext-0.5.7/lib.rs:113:24
    |
113 |     cx.span_err(parser.span, &err);
    |                        ^^^^ unknown field
    |
    = note: available fields are: `sess`, `token`, `prev_span`, `recurse_into_file_modules`, `root_module_name`, `cfg_mods`

error: aborting due to 8 previous errors

Some errors have detailed explanations: E0425, E0432, E0599, E0609.
For more information about an error, try `rustc --explain E0425`.
error: Could not compile `peg-syntax-ext`.
warning: build failed, waiting for other jobs to finish...
error: build failed

Process finished with exit code 101
```

**Comment by kevinmehall** [comment]
peg-syntax-ext was built on the deprecated nighly-only compiler plugin API which is no longer supported. Try out the `master` branch of rust-peg, which switches to the `proc-macro` API, which works on stable `rustc`, and supports better syntax and error messages. This will be released soon as 0.6 or 1.0, but if you want to stick with 0.5 for now, you can use the [build.rs](https://github.com/kevinmehall/rust-peg/tree/0.5#with-a-build-script) method.

**Comment by kevinmehall** [comment]
[Released 0.6.0](https://github.com/kevinmehall/rust-peg/releases/tag/0.6.0).
